list of users.py

```diff
@@ -0,0 +1,54 @@
+import urllib.request
+import json
+import pandas as pd
+
+
+f=open(r"E:\CC rank predictor\New Text Document.txt", "w",encoding="utf-8")
+a1="https://www.codechef.com/api/ratings/all?page="
+b1="&sortBy=rating&order=desc&itemsPerPage=100"
+flag=1
+t=1
+
+#making of headers
+html_file1=urllib.request.Request("https://www.codechef.com/api/ratings/all?page=1&sortBy=rating&order=desc&itemsPerPage=100", headers={"User-Agent" :"Mozilla /5.0"})
+with urllib.request.urlopen(html_file1) as url1:
+    data1 = json.loads(url1.read().decode())
+#    print(data1)
+data1=data1["list"]
+attributes=list(data1[0].keys())
+attributes.pop()
+headers=""
+for i in range(len(attributes)):
+    
+    headers=headers+attributes[i]+","
+f.write(headers+"\n")
+
+
+
+
+err=0
+
+
+while(t<4420):
```

Comment:
You should not hardcode number of pages. The number of page is available in the `availablePages` field in the JSON returned from the API.